editor/src/components/canvas/dom-walker.ts

```diff
@@ -231,14 +279,18 @@ export function useDomWalker(props: CanvasContainerProps): React.Ref<HTMLDivElem
   const containerRef = React.useRef<HTMLDivElement>(null)
   const rootMetadataInStateRef = useRefEditorState((store) => store.editor.domMetadataKILLME)
   const invalidatedSceneIDsRef = React.useRef<Set<string>>(emptySet())
+  const invalidatedPathsForStylesheetCacheRef = React.useRef<Set<string>>(new Set())
```

Comment:
Just as a safeguard.
```suggestion
  const invalidatedPathsForStylesheetCacheRef = React.useRef<Set<string>>(emptySet())
```